msal-auth-node/app.js

```diff
@@ -0,0 +1,38 @@
+/*
+ This application demonstrates how to include
+ authorization for an API endpoint in Node.Js 16.
+*/
+
+// Node.Js Express Framework
+const express = require('express')
+
+// Libraries used to validate authentication tokens
+const jwt = require('express-jwt')
+const jwks = require('jwks-rsa')
+const jwtAuthz = require('express-jwt-authz')
+
+const app = express()
+
+app.use(jwt({
+   secret: jwks.expressJwtSecret({
+   
+   // Full URL, in the form of: https://login.microsoftonline.com/<tenant>/discovery/v2.0/keys
+   jwksUri: ''
+  }),
```

Comment:
Fix indent.  Did you run the formatter on this?

```suggestion
   secret: jwks.expressJwtSecret({
     // Full URL, in the form of: https://login.microsoftonline.com/<tenant>/discovery/v2.0/keys
     jwksUri: ''
  }),
```

Reply:
I did - but I may have made some changes after the fact. Will check w the formatter again.